01week/datatypes.js

```diff
@@ -0,0 +1,75 @@
+const displayDayTime = () =>{
+    const d = new Date();
+    const day = d.getDay();
+    switch(day){
+      case 1:
+        console.log('Today is Monday');
+        break;
+      case 2:
+        console.log('Today is Tuesday');
+        break;
+      case 3:
+        console.log('Today is Wednesday');
+        break;
+      case 4:
+        console.log('Today is Thursday');
+        break;
+      case 5:
+        console.log('Today is Friday');
+        break;
+      case 6:
+        console.log('Today is Saturday');
+        break;
+      case 7:
+        console.log('Today is Sunday');
+        break;
+      default:
+        break;
+    }
+    
+    const hour   = d.getHours();
+    const minute = d.getMinutes();
+    const second = d.getSeconds();
+    console.log('Time now is:',hour,':',minute,':',second);
+    }
```

Comment:
fix the indentation here, this bracket shouldn't be indented, it should line up with the opening bracket